liquid.py

```diff
@@ -256,17 +285,50 @@ def alloc(job, group):
     print(first(running, 'running allocations'))
 
 
-def nomad_address():
+def get_search_collections():
+    try:
+        return docker_exec('hoover-search', './manage.py', 'listcollections').split()
+    except CalledProcessError as e:
+        print(e.output.decode(), file=sys.stderr)
+        raise
+
+
+def initcollection(name):
+    """Initialize collection with given name.
+    
+    Create the snoop database, create the search index, run dispatcher, add collection
+    to search.
+    
+    :param name: the collection name
+    """
+    if name not in config.collections:
+        raise RuntimeError('Collection %s does not exist in the liquid.ini file.', name)
+
+    if name in get_search_collections():
+        log.info(f'Collection "{name}" was already initialized.')
+        return
+
+    shell(f'snoop-{name}-api', './manage.py', 'initcollection')
+
+    shell('hoover-search', './manage.py', 'addcollection', name, '--index',
+          name, f'{get_nomad_address()}:8765/{name}/collection/json', '--public')
+
+
+def get_nomad_address():
     """
     Print the nomad server's address.
     """
     members = [m['Addr'] for m in nomad.agent_members()]
-    print(first(members, 'members'))
+    return first(members, 'members')
+
+
+def nomad_address():
+    print(get_nomad_address())
 
 
 def set_collection_defaults(name, settings):
     settings['name'] = name
-    settings.setdefault('workers', '1')
+    settings.setdefault('workers', 1)
```

Comment:
And now it's a number instead of a string. Why?

Reply:
My mistake, forgot it has to be a string.